2025-03-02 17:37:33,428 DEBUG: 分类 '科技' 的文章数量: 0
None
2025-03-02 17:37:33,429 DEBUG: 分类 '小说' 的文章数量: 1
None
2025-03-02 17:37:33,429 DEBUG: 分类 '新闻' 的文章数量: 0
None
2025-03-02 17:37:33,431 DEBUG: 标签 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:33,433 DEBUG: 标签 'AI' 的文章数量: 1
None
2025-03-02 17:37:33,434 DEBUG: 标签 'Python' 的文章数量: 2
None
2025-03-02 17:37:33,436 DEBUG: 标签 '励志' 的文章数量: 1
None
2025-03-02 17:37:33,437 DEBUG: 标签 'test-labe01' 的文章数量: 1
None
2025-03-02 17:37:33,439 DEBUG: 标签 'test' 的文章数量: 0
None
2025-03-02 17:37:33,440 INFO: 成功获取侧边栏数据: 6个分类, 6个标签, 4篇最新文章
None
2025-03-02 17:37:33,735 INFO: 访问博客首页
None
2025-03-02 17:37:33,735 DEBUG: 查询参数: page=1, search=, category_id=None, tag_id=None
None
2025-03-02 17:37:33,737 INFO: 成功获取文章列表，共4篇文章
None
2025-03-02 17:37:33,737 INFO: 获取侧边栏数据
None
2025-03-02 17:37:33,738 DEBUG: 分类 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:33,739 DEBUG: 分类 'AI' 的文章数量: 0
None
2025-03-02 17:37:33,740 DEBUG: 分类 '编程' 的文章数量: 1
None
2025-03-02 17:37:33,740 DEBUG: 分类 '科技' 的文章数量: 0
None
2025-03-02 17:37:33,741 DEBUG: 分类 '小说' 的文章数量: 1
None
2025-03-02 17:37:33,742 DEBUG: 分类 '新闻' 的文章数量: 0
None
2025-03-02 17:37:33,744 DEBUG: 标签 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:33,746 DEBUG: 标签 'AI' 的文章数量: 1
None
2025-03-02 17:37:33,747 DEBUG: 标签 'Python' 的文章数量: 2
None
2025-03-02 17:37:33,751 DEBUG: 标签 '励志' 的文章数量: 1
None
2025-03-02 17:37:33,754 DEBUG: 标签 'test-labe01' 的文章数量: 1
None
2025-03-02 17:37:33,756 DEBUG: 标签 'test' 的文章数量: 0
None
2025-03-02 17:37:33,757 INFO: 成功获取侧边栏数据: 6个分类, 6个标签, 4篇最新文章
None
2025-03-02 17:37:35,199 INFO: 访问博客首页
None
2025-03-02 17:37:35,200 DEBUG: 查询参数: page=1, search=, category_id=None, tag_id=1
None
2025-03-02 17:37:35,201 DEBUG: 应用标签过滤: 1
None
2025-03-02 17:37:35,202 INFO: 成功获取文章列表，共2篇文章
None
2025-03-02 17:37:35,203 INFO: 获取侧边栏数据
None
2025-03-02 17:37:35,203 DEBUG: 分类 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:35,204 DEBUG: 分类 'AI' 的文章数量: 0
None
2025-03-02 17:37:35,205 DEBUG: 分类 '编程' 的文章数量: 1
None
2025-03-02 17:37:35,205 DEBUG: 分类 '科技' 的文章数量: 0
None
2025-03-02 17:37:35,206 DEBUG: 分类 '小说' 的文章数量: 1
None
2025-03-02 17:37:35,206 DEBUG: 分类 '新闻' 的文章数量: 0
None
2025-03-02 17:37:35,208 DEBUG: 标签 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:35,209 DEBUG: 标签 'AI' 的文章数量: 1
None
2025-03-02 17:37:35,211 DEBUG: 标签 'Python' 的文章数量: 2
None
2025-03-02 17:37:35,212 DEBUG: 标签 '励志' 的文章数量: 1
None
2025-03-02 17:37:35,213 DEBUG: 标签 'test-labe01' 的文章数量: 1
None
2025-03-02 17:37:35,214 DEBUG: 标签 'test' 的文章数量: 0
None
2025-03-02 17:37:35,215 INFO: 成功获取侧边栏数据: 6个分类, 6个标签, 4篇最新文章
None
2025-03-02 17:37:35,418 WARNING: Not Found: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
None
2025-03-02 17:37:35,530 INFO: 访问博客首页
None
2025-03-02 17:37:35,531 DEBUG: 查询参数: page=1, search=, category_id=None, tag_id=1
None
2025-03-02 17:37:35,533 DEBUG: 应用标签过滤: 1
None
2025-03-02 17:37:35,536 INFO: 成功获取文章列表，共2篇文章
None
2025-03-02 17:37:35,536 INFO: 获取侧边栏数据
None
2025-03-02 17:37:35,538 DEBUG: 分类 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:35,539 DEBUG: 分类 'AI' 的文章数量: 0
None
2025-03-02 17:37:35,539 DEBUG: 分类 '编程' 的文章数量: 1
None
2025-03-02 17:37:35,540 DEBUG: 分类 '科技' 的文章数量: 0
None
2025-03-02 17:37:35,541 DEBUG: 分类 '小说' 的文章数量: 1
None
2025-03-02 17:37:35,541 DEBUG: 分类 '新闻' 的文章数量: 0
None
2025-03-02 17:37:35,543 DEBUG: 标签 '个人博客' 的文章数量: 2
None
2025-03-02 17:37:35,544 DEBUG: 标签 'AI' 的文章数量: 1
None
2025-03-02 17:37:35,546 DEBUG: 标签 'Python' 的文章数量: 2
None
2025-03-02 17:37:35,547 DEBUG: 标签 '励志' 的文章数量: 1
None
2025-03-02 17:37:35,549 DEBUG: 标签 'test-labe01' 的文章数量: 1
None
2025-03-02 17:37:35,551 DEBUG: 标签 'test' 的文章数量: 0
None
2025-03-02 17:37:35,552 INFO: 成功获取侧边栏数据: 6个分类, 6个标签, 4篇最新文章
None
2025-03-02 17:37:35,784 WARNING: Not Found: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
None
2025-03-02 17:37:44,344 ERROR: 登录过程中发生错误: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
Traceback (most recent call last):
  File "/data/myblog/app/controllers/auth/__init__.py", line 102, in login
    return render_template('auth/login.html', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/data/myblog/app/templates/auth/login.html", line 1, in top-level template code
    {% extends "blog/layout.html" %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/blog/layout.html", line 22, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/auth/login.html", line 34, in block 'content'
    <a href="{{ url_for('auth.register') }}">没有账号？立即注册</a>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1074, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1063, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 919, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?

None
2025-03-02 17:37:44,346 ERROR: Exception during request: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
None
2025-03-02 17:37:44,503 ERROR: 登录过程中发生错误: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
Traceback (most recent call last):
  File "/data/myblog/app/controllers/auth/__init__.py", line 102, in login
    return render_template('auth/login.html', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/data/myblog/app/templates/auth/login.html", line 1, in top-level template code
    {% extends "blog/layout.html" %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/blog/layout.html", line 22, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/auth/login.html", line 34, in block 'content'
    <a href="{{ url_for('auth.register') }}">没有账号？立即注册</a>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1074, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1063, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 919, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?

None
2025-03-02 17:37:44,506 ERROR: Exception during request: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
None
